docs/dsp-tools-excel2xml.md

```diff
@@ -5,30 +5,66 @@ dsp-tools assists you in converting a data source in CSV/XLS(X) format to an XML
 transformation from Excel/CSV to XML: 
 
  - The CLI command `dsp-tools excel2xml` creates an XML file from an Excel/CSV file which is already structured 
-   according to the DSP specifications. This is mostly used for DaSCH-interal data migration. The CLI command is 
-   documented [here](dsp-tools-excel.md#cli-command-excel2xml).
+   according to the DSP specifications. This is mostly used for DaSCH-interal data migration. **The CLI command is 
+   documented [here](dsp-tools-excel.md#cli-command-excel2xml).**
  - The module `excel2xml` can be imported into a custom Python script that transforms any tabular data into an XML. This
-   use case is more frequent, because data from research projects have a variety of formats/structures. **This document
-   only treats the `excel2xml` module.**
-
-
-## How to use the module excel2xml
-At the end of this document, you find a sample Python script. In the following, it is explained how to use it.
-
-
-### General preparation
-Insert your ontology name, project shortcode, and the path to your data source. If necessary, activate one of the lines
-that are commented out.  
-Then, the `root` element is created, which represents the `<knora>` tag of the XML document. As first children of 
+   use case is more frequent, because data from research projects have a variety of formats/structures. **The 
+   `excel2xml` module is documented on this page.**
+
+
+# How to use the module excel2xml
+Save the following files into a directory, and run the Python script: 
+
+ - sample data: [excel2xml_sample_data.csv](./assets/templates/excel2xml_sample_data.csv)
+ - sample ontology: [excel2xml_sample_onto.json](./assets/templates/excel2xml_sample_onto.json)
+ - sample script: [excel2xml_sample_script.py](./assets/templates/excel2xml_sample_script.py)
+
+This is the simplified pattern how the Python script works:
+
+```python
+1  main_df = pd.read_csv("excel2xml_sample_data.csv", dtype="str", sep=",")
+2  root = excel2xml.make_root(...)
+3  root = excel2xml.append_permissions(root)
+4  # if necessary: create list mappings, according to explanation below
+5  for index, row in main_df.iterrows():
+6      resource = excel2xml.make_resource(...)
+7      resource.append(excel2xml.make_text_prop(...))
+8      root.append(resource)
+9  excel2xml.write_xml(root, "data.xml")
+```
+```
+1 read in your data source with the pandas library (https://pandas.pydata.org/)  
+2 create the root element `<knora>`  
+3 append the permissions  
+4 if necessary: create list mappings (see below)  
+5 iterate through the rows of your data source:  
+6   create the `<resource>` tag  
+7   append properties to it  
+8   append the resource to the root tag `<knora>`  
+9 save the finished XML file  
+```
+
+<br>
```

Comment:
Yes, you mentioned it before, and I agree. 
Do I still have an influence on the mkdocs configuration, now that we have moved to docs.dasch.swiss?
I will think of another way how to get around this hack. I fear that the sentence after the <br> doesn't get enough attention if it is too close to the  code block. But perhaps there are other ways how to achieve that.

Reply:
we have a docs repo on github, where changes to mkdocs can be made. You may want to check with Marcin and Mike when changing something there, as they are now mostly responsible for that repo because they do deployments, which includes deploying docs.

To highlight this sentence, you could make it a quote (by adding `>` in front of it, and see how it looks in mkdocs (or leave the `<br>` in for once... it doesn't hurt anyone.